docs/docsite/rst/user_guide/playbooks_conditionals.rst

```diff
@@ -190,8 +192,19 @@ tasks within the included file(s). A common situation where this distinction is
     - debug:
         var: x
 
-In the above example, if ``import_tasks`` had been used instead both included tasks would have also been skipped. With ``include_tasks``
-instead, the tasks are executed as expected because the conditional is not applied to them.
+This expands at include time to the equivalent of::
+
+    - set_fact:
+        x: foo
+      when: x is not defined
+    - debug:
+        var: x
+      when: x is not defined
+
+Thus if ``x`` is initially undefined, the ``debug`` task will be skipped.  By using ``include_tasks`` instead of ``import_tasks``,
+both tasks from ``other_tasks.yml`` will be executed as expected.
+
+For more information on the differences between ``include`` v ``importing`` see :ref:`playbooks_reuse`.
```

Comment:
typo? I'd make this "the differences between `include` and `import` . . . "

Reply:
Done